core/utils/force-locale.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace Elementor\Core\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Force translation to use a specific locale.
+ *
+ * A hacky class to force any translation functions in the call-stack between the
+ * `force()` & `reset()` methods to use a specific locale.
+ */
+class Force_Locale {
```

Comment:
Maybe add some doc block to the properties and the methods?

Reply:
Done